docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -1,77 +1,94 @@
-| `ASSET_KEY+`          | Selects an asset and one layer downstream of the asset. Including multiple `+`s will select that number of downstream layers from the asset. Any number of `+`s is supported. [See an example](#specific-downstream). |
+Filtering by upstream and downstream layers can help you understand the data flow within your assets.
 
-## Examples
+- **Upstream assets** provide input to a given asset.
+- **Downstream assets** receive input from a given asset.
+- A **layer** is a level of connection in the graph. One layer up or down means a direct connection, "two layers" means connections two steps away, and so on.
 
-The examples in this section use the following asset graph from the [Dagster University Essentials project](https://github.com/dagster-io/project-dagster-university) to demonstrate how to use the selection syntax:
+| Asset selection | Syntax | Description |
+|------|--------|-------------|
+| **All layers upstream of an asset** | `+key:"my_asset"` | Selects all upstream assets that provide input into `my_asset`. |
+| **One layer upstream of an asset** | `1+key:"my_asset"` | Selects assets that directly provide input to `my_asset`. |
+| **Two layers upstream of an asset** | `2+key:"my_asset"` | Selects assets two steps upstream from `my_asset`. |
+| **All layers downstream of an asset** | `key:"my_asset"+` | Selects all downstream assets that depend on `my_asset`. |
+| **One layer downstream of an asset** | `key:"my_asset"+1` | Selects assets that directly receive input from `my_asset`. |
+| **Two layers downstream of an asset** | `key:"my_asset"+2` | Selects assets two steps downstream from `my_asset`. |
+| **One layer upstream and downstream of an asset** | `1+key:"my_asset"+1` | Selects one layer of assets providing input to and receiving input from `my_asset`. |
+| **Two layers upstream and downstream of an asset** | `2+key:"my_asset"+2` | Selects two layers of assets providing input to and receiving input from `my_asset`. |
+| **All layers upstream and downstream of an asset** | `+key:"my_asset"+ ` | Selects all assets upstream and downstream of `my_asset`. |
 
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/asset-selection-syntax-dag.png)
+## Filters, operands, and grouping
 
-### Selecting a single asset \{#single-asset}
+Filters allow you to narrow your asset selection using specific criteria.
 
-To select a single asset, use the asset's asset key. This example selects the `taxi_zones_file` asset:
+| Filter | Syntax | Description | Supported views |
+|--------|--------|-------------|-----------------|
+| **Exact key** | `key:"my_key"` | Selects assets with the exact key `my_key`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Key substring with wildcard filter** | `key:partial_key_*`| Selects assets whose key contains `partial_key`. | OSS, Dagster+, Dagster+ branch deployments |
```

Comment:
```suggestion
| **Key with wildcard filter** | `key:partial_key_*`| Selects assets whose key starts with `partial_key_`. | OSS, Dagster+, Dagster+ branch deployments |
```

Reply:
Incorporated into my most recent set of changes